server/utils/urlGenerator.ts

```diff
@@ -309,9 +309,11 @@ const adjudicationUrls = {
     root: '/print',
     matchers: {
       dis12: '/:chargeNumber/dis12/pdf',
+      dis5: '/:chargeNumber/dis5/pdf',
     },
     urls: {
       dis12: (chargeNumber: string) => `${adjudicationUrls.printPdf.root}/${chargeNumber}/dis12/pdf`,
+      dis5: (chargeNumber: string) => `${adjudicationUrls.printPdf.root}/${chargeNumber}/dis5/pdf`,
```

Comment:
Endpoint is exposed for testing - `<adjudicationsUrl>/print/1525796/dis5/pdf` - nothing is linking to it yet, should be ok?